honeybee_schema/radiance/properties.py

```diff
@@ -0,0 +1,173 @@
+                    'the contribution of individual objects).'
+        )
+
+
+class _PropertiesBase(IDdBaseModel):
+    """Base class of Radiance Properties."""
+
+    modifier: ModifierBase = Field(
+        default=None,
+        description='A Honeybee Radiance Modifier for the object.'
+        )
+
+    modifier_blk: ModifierBase = Field(
+        default=None,
+        description='A Honeybee Radiance Modifier to be used for this object'
+                    'in direct solar simulations and in isolation studies (assessing'
+                    'the contribution of individual objects).'
+        )
+
+
+class ApertureRadiancePropertiesAbridged(_PropertiesBase):
+    """Radiance Properties for Honeybee Aperture Abridged."""
+
+    type: constr(regex='^ApertureRadiancePropertiesAbridged$') = \
+        'ApertureRadiancePropertiesAbridged'
+
+
+class ApertureRadianceProperties(_PropertiesBase):
+    """Radiance Properties for Honeybee Aperture."""
+
+    type: constr(regex='^ApertureRadianceProperties$') = 'ApertureRadianceProperties'
+
+
+class DoorRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Door Abridged."""
+
+    type: constr(regex='^DoorRadiancePropertiesAbridged$') = \
+        'DoorRadiancePropertiesAbridged'
+
+
+class DoorRadianceProperties(_PropertiesBase):
+    """Radiance Properties for Honeybee Door."""
+
+    type: constr(regex='^DoorRadianceProperties$') = 'DoorRadianceProperties'
+
+
+class FaceRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Face Abridged."""
+
+    type: constr(regex='^FaceRadiancePropertiesAbridged$') = \
+        'FaceRadiancePropertiesAbridged'
+
+
+class FaceRadianceProperties(_PropertiesBase):
+    """Radiance Properties for Honeybee Face."""
+
+    type: constr(regex='^FaceRadianceProperties$') = 'FaceRadianceProperties'
+
+
+class ShadeRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Shade Abridged."""
+
+    type: constr(regex='^ShadeRadiancePropertiesAbridged$') = \
+        'ShadeRadiancePropertiesAbridged'
+
+
+class ShadeRadianceProperties(_PropertiesBase):
+    """Radiance Properties for Honeybee Shade."""
+
+    type: constr(regex='^ShadeRadianceProperties$') = 'ShadeRadianceProperties'
+
+
+class ModelRadianceProperties(IDdBaseModel):
+    """Radiance Properties for Honeybee Model."""
+
+    type: constr(regex='^ModelRadianceProperties$') = 'ModelRadianceProperties'
+
+
+    # TODO: Clarification: does there need to be an abridged version of all these lists?
+    modifiers: List[
+        ModifierBase
+    ] = Field(
+        default=[],
+        description='A list of all unique modifiers in the model. '
+                    'This includes modifiers across all Faces, Apertures, Doors, Shades, '
+                    'Room ModifierSets, and the global_modifier_set. (default: []).'
+        )
+
+    blk_modifiers: List[
```

Comment:
This property should be deleted. `ModelRadianceProperties` don't have a key for `blk_modifiers`. Follow the example here:
https://github.com/ladybug-tools/honeybee-schema/blob/master/samples/model/model_complete_multiroom_radiance.json#L410-L636

Reply:
@chriswmackey the ModelRadianceProperties class seems to have a property for blk_modifiers: https://github.com/ladybug-tools/honeybee-radiance/blob/master/honeybee_radiance/properties/model.py#L24-L33

Or should I be using the sample json as the source of truth for schemas?